lib/vfscore/Config.uk

```diff
@@ -77,13 +82,14 @@ if LIBVFSCORE_AUTOMOUNT_ROOTFS
 	default "ramfs" if LIBVFSCORE_ROOTFS_RAMFS
 	default "9pfs" if LIBVFSCORE_ROOTFS_9PFS
 	default "initrd" if LIBVFSCORE_ROOTFS_INITRD
+	default "einitrd" if LIBVFSCORE_ROOTFS_EINITRD
 	default LIBVFSCORE_ROOTFS_CUSTOM_ARG if LIBVFSCORE_ROOTFS_CUSTOM
 	default ""
 
 	# The root device option is hidden for RamFS and 9PFS
 	config LIBVFSCORE_ROOTDEV
 	string "Default root device"
-	depends on !LIBVFSCORE_ROOTFS_RAMFS && !LIBVFSCORE_ROOTFS_INITRD
+	depends on !LIBVFSCORE_ROOTFS_RAMFS && !LIBVFSCORE_ROOTFS_INITRD && !LIBVFSCORE_ROOTFS_EINITRD
```

Comment:
The list is getting a bit longer. Can we instead depend on a hidden `LIBVFSCORE_HAVE_ROOTDEV` option that's selected by `LIBVFSCORE_ROOTFS_9PFS`. (+ similar options for `LIBVFSCORE_ROOTFLAGS` etc.)

Reply:
In order to avoid repeating the same `depends on` statement for 3 times, I am going to encapsulate the options under an `if` condition.